site/content/docs/alerts/10004.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Tech Detected - Apache HTTP Server"
+alertid: 10004
+alertindex: 1000400
+alerttype: "Tool"
+alertcount: 1
+status: release
+type: alert
+risk: Informational
+solution: ""
+references:
+   - https://httpd.apache.org
+other: "The following CPE is associated with the identified tech: cpe:2.3:a:apache:http_server:*:*:*:*:*:*:*:* The following version(s) is/are associated with the identified tech: 2.4.7"
+cwe: 200
+wasc: 13
+alerttags: 
+  - CWE-200
+code: https://github.com/zaproxy/zap-extensions/blob/main/addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/ExtensionWappalyzer.java
```

Comment:
This will need changes.

Reply:
Yeah, access control alerts are the same (linking the extension, not using what's actually set in the code)